tests/test_store.py

```diff
@@ -483,6 +480,65 @@ def test_get_files_metadata_from_dataset(self, synapse_store):
                     "entityId": ["syn123", "syn456"],
                 }
 
+    async def test_get_async_annotation(self, synapse_store: SynapseStorage) -> None:
+        """test get annotation async function"""
+        mock_syn_id = "syn1234"
+
+        with patch(
+            "schematic.store.synapse.get_entity_id_bundle2",
+            new_callable=AsyncMock,
+            return_value="mock",
+        ) as mock_get_entity_id_bundle2:
+            mock_get_entity_id_bundle2.return_value = "mock"
+            result = await synapse_store.get_async_annotation(synapse_id=mock_syn_id)
+
+            mock_get_entity_id_bundle2.assert_called_once_with(
+                entity_id=mock_syn_id,
+                request={"includeAnnotations": True},
+                synapse_client=synapse_store.syn,
+            )
+            assert result == "mock"
+
+    async def test_store_async_annotation(self, synapse_store: SynapseStorage) -> None:
+        """test store annotations async function"""
+        annos_dict = {
+            "annotations": {
+                "id": "mock_syn_id",
+                "etag": "mock etag",
+                "annotations": {
+                    "Id": {"type": "STRING", "value": ["mock value"]},
+                    "EntityId": {"type": "STRING", "value": ["mock_syn_id"]},
+                    "SampleID": {"type": "STRING", "value": [""]},
+                    "Component": {"type": "STRING", "value": ["mock value"]},
+                },
+            },
+            "FileFormat": "mock format",
+            "Component": "mock component",
+            "Id": "mock_string",
+            "EntityId": "mock_id",
+        }
+        expected_dict = Annotations(
+            annotations={
+                "Id": ["mock_string"],
+                "EntityId": ["mock_syn_id"],
+                "SampleID": [""],
+                "Component": ["mock value"],
+                "FileFormat": ["mock_format"],
+            },
+            etag="mock etag",
+            id="mock syn_id",
+        )
+
+        with patch(
+            "schematic.store.synapse.Annotations.store_async",
+            new_callable=AsyncMock,
+            return_value=expected_dict,
+        ) as mock_store_async:
+            result = await synapse_store.store_async_annotation(annos_dict)
```

Comment:
Should we also try to assert that result.annotations is equal to expected_dict.annotations here?

Reply:
can I just add: `assert result==expected_dict` instead?

Reply:
idk - can you? Does it pass if you use that assert?

Reply:
Yes. That works!